actors/user/src/test/java/org/sunbird/user/UserStatusActorTest.java

```diff
@@ -0,0 +1,156 @@
+import org.keycloak.admin.client.resource.UserResource;
+import org.keycloak.admin.client.resource.UsersResource;
+import org.keycloak.representations.idm.UserRepresentation;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.sunbird.cassandraimpl.CassandraOperationImpl;
+import org.sunbird.common.exception.ProjectCommonException;
+import org.sunbird.common.models.response.Response;
+import org.sunbird.common.models.util.ActorOperations;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.models.util.KeyCloakConnectionProvider;
+import org.sunbird.common.request.Request;
+import org.sunbird.common.responsecode.ResponseCode;
+import org.sunbird.helper.ServiceFactory;
+import org.sunbird.models.user.User;
+import org.sunbird.user.actors.UserStatusActor;
+import org.sunbird.user.service.UserService;
+import org.sunbird.user.service.impl.UserServiceImpl;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({UserServiceImpl.class, KeyCloakConnectionProvider.class, ServiceFactory.class})
+@PowerMockIgnore("javax.management.*")
+public class UserStatusActorTest {
+
+  private static final Props props = Props.create(UserStatusActor.class);
+  private static final ActorSystem system = ActorSystem.create("system");
+  private static final String userId = "someUserId";
+  private User user;
+  private static CassandraOperationImpl cassandraOperation;
+
+  @Before
+  public void init() {
+
+    PowerMockito.mockStatic(ServiceFactory.class);
+    UserRepresentation ur = mock(UserRepresentation.class);
+    RealmResource realmResource = mock(RealmResource.class);
+    cassandraOperation = mock(CassandraOperationImpl.class);
+
+    when(ServiceFactory.getInstance()).thenReturn(cassandraOperation);
+    Keycloak keycloak = mock(Keycloak.class);
+
+    PowerMockito.mockStatic(UserServiceImpl.class);
+    UserService userService = mock(UserService.class);
+    when(UserServiceImpl.getInstance()).thenReturn(userService);
+
+    PowerMockito.mockStatic(KeyCloakConnectionProvider.class);
+    when(KeyCloakConnectionProvider.getConnection()).thenReturn(keycloak);
+    when(keycloak.realm(Mockito.anyString())).thenReturn(realmResource);
+
+    UsersResource usersResource = mock(UsersResource.class);
+    when(realmResource.users()).thenReturn(usersResource);
+
+    UserResource resource = mock(UserResource.class);
+    when(usersResource.get(Mockito.any())).thenReturn(resource);
+    when(resource.toRepresentation()).thenReturn(ur);
+    ur.setEnabled(Mockito.anyBoolean());
+    user = mock(User.class);
+    when(userService.getUserById(Mockito.anyString())).thenReturn(user);
+  }
+
+  @Test
+  public void testBlockUserSuccess() {
+
+    TestKit probe = new TestKit(system);
```

Comment:
all tests have same code pattern -> it seems -> can be converted to reusable single method taking argument - > whether (boolean isDeleted, ActorOperation operation, Response expectedResponse)